host/sgx/quote.h

```diff
@@ -71,6 +71,56 @@ oe_result_t oe_verify_report_internal(
     size_t report_size,
     oe_report_t* parsed_report);
 
+/*
+**==============================================================================
+**
+** sgx_get_supplemental_data_size()
+**
+**==============================================================================
+*/
+oe_result_t sgx_get_supplemental_data_size(
+    const oe_uuid_t* format_id,
+    const void* opt_params,
```

Comment:
"optional_parameters"?  Same question for line 96.

Reply:
Same as above.